client/me/purchases/cancel-purchase/index.jsx

```diff
@@ -209,49 +245,113 @@ class CancelPurchase extends Component {
-				</CompactCard>
-				<CompactCard className="cancel-purchase__footer">
-					<div className="cancel-purchase__refund-amount">
-						{ this.renderFooterText( this.props ) }
+					<Gridicon icon="chevron-left" size={ 18 } />
+					{ this.props.translate( 'Back' ) }
+				</Button>
+
+				<FormattedHeader
+					className="cancel-purchase__formatter-header"
+					brandFont
+					headerText={ heading }
+					align="left"
+				/>
+
+				<div className="cancel-purchase__layout">
+					<div className="cancel-purchase__layout-col cancel-purchase__layout-col-left">
+						<PurchaseSiteHeader siteId={ siteId } name={ siteName } domain={ siteDomain } />
+
+						<CancelPurchaseRefundInformation
+							purchase={ purchase }
+							isJetpackPurchase={ isJetpackPurchase }
+							confirmBundledDomain={ this.state.confirmCancelBundledDomain }
+							cancelBundledDomain={ this.state.cancelBundledDomain }
+							onCancelConfirmationStateChange={ this.onCancelConfirmationStateChange }
+							site={ this.props.site }
+						/>
+
+						{ isSubscription( purchase ) && (
+							<Fragment>
+								<div className="purchase-cancel-buttons">
+									<CancelPurchaseButton
+										purchase={ purchase }
+										includedDomainPurchase={ this.props.includedDomainPurchase }
+										disabled={
+											this.state.cancelBundledDomain && ! this.state.confirmCancelBundledDomain
+										}
+										siteSlug={ this.props.siteSlug }
+										cancelBundledDomain={ this.state.cancelBundledDomain }
+										purchaseListUrl={ this.props.purchaseListUrl }
+										getConfirmCancelDomainUrlFor={ this.props.getConfirmCancelDomainUrlFor }
+										activeSubscriptions={ this.getActiveMarketplaceSubscriptions() }
+									/>
+
+									<Button
+										className="purchase-cancel-keep-plan"
+										href={ this.props.getManagePurchaseUrlFor(
+											this.props.siteSlug,
+											this.props.purchaseId
+										) }
+									>
+										{ this.keepButtonCTA( locale ) }
+									</Button>
+								</div>
+
+								<p className="cancel-purchase--support-link cancel-purchase--support-link--main">
+									{ this.supportLink() }
+								</p>
+							</Fragment>
+						) }
+
+						{ ! isSubscription( purchase ) && (
+							<Fragment>
+								<CompactCard className="cancel-purchase__product-information">
+									<div className="cancel-purchase__purchase-name">{ purchaseName }</div>
+									<div className="cancel-purchase__description">{ purchaseType( purchase ) }</div>
+									<ProductLink purchase={ purchase } selectedSite={ this.props.site } />
+								</CompactCard>
+
+								<CompactCard className="cancel-purchase__footer">
+									<div className="cancel-purchase__refund-amount">
+										{ renderFooterText(
+											this.state,
+											this.props.translate,
+											this.props.moment,
+											this.includedDomainPurchase,
+											purchase
```

Comment:
I don't see a `includedDomainPurchase` property in this component. Did you mean `this.props.includedDomainPurchase`?

Reply:
`includedDomainPurchase` has been replaced with `this.props.includedDomainPurchase`